src/components/backend-ai-console.ts

```diff
@@ -716,6 +721,22 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
       document.dispatchEvent(event);
     }
   }
+  _readRecentProjectGroup() {
+    let value: string | null = localStorage.getItem('backendaiconsole.projectGroup');
+    if (value !== null && value !== '' && value !== '""') {
```

Comment:
I think the condition here can be simplified, unless you want explicit comparison, since `undefined`, `null`, `''`, and `""` are all evaluated as `false`:

```javascript
if (value) {
```

Be careful, however, `[]` and `{}` are evaluated as `true` in `if` statement.

If you want a compact expression here, it is also possible to use the ternary operator.

```javascript
return value ? value : this.backendaiclient.current_group;
```

Reply:
I guess there's a scarce chance of getting ```[]```  or ```{}``` type in this case, but I'll try to catch these values as well.